crates/storage/libmdbx-rs/src/transaction.rs

```diff
@@ -108,7 +106,7 @@ where
     /// The caller **must** ensure that the pointer is not used after the
     /// lifetime of the transaction.
```

Comment:
this comment is probably outdated now, could be good to add smthg like "caller is responsible for aborting timed out transactions"